ocaml/typing/typetexp.ml

```diff
@@ -715,34 +642,17 @@ let transl_bound_vars : (_, _) Either.t -> _ =
   | Right vars_jkinds -> TyVarEnv.make_poly_univars_jkinds
                            ~context:(fun v -> Univar ("'" ^ v)) vars_jkinds
 
-let rec transl_type env ~policy ?(aliased=false) ~row_context mode styp =
-  Builtin_attributes.warning_scope styp.ptyp_attributes
-    (fun () -> transl_type_aux env ~policy ~aliased ~row_context mode styp)
-||||||| 121bedcfd2
-let rec transl_type env policy styp =
-  Builtin_attributes.warning_scope styp.ptyp_attributes
-    (fun () -> transl_type_aux env policy styp)
-=======
-(* Forward declaration (set in Typemod.type_open) *)
```

Comment:
I think you dropped this comment on `type_open`.

Reply:
Indeed, thanks.